e2e/tests/replica_resiliency_test.robot

```diff
@@ -1,19 +1,26 @@
 *** Settings ***
-Documentation    Negative Test Cases
+Documentation       Negative Test Cases
 
-Resource    ../keywords/volume.resource
+Resource            ../keywords/common.resource
+Resource            ../keywords/replica.resource
+Resource            ../keywords/volume.resource
 
-Test setup    set_test_name    ${TEST NAME}
+Suite Setup         set_test_suite_environment
+Test Setup          set_test_environment    ${TEST NAME}
+Test Teardown       Cleanup resource and resume state
 
-Test Teardown    cleanup_resources
 
 *** Variables ***
-${LOOP_COUNT}    1
+${LOOP_COUNT}           1
+${volume_size_gb}=      2
+${volume_type}=         RWO
+
 
 *** Test Cases ***
 Replica Rebuilding While Replica Deletion
-    Create a volume 2 GB with 3 replicas
-    Write data to the volume
+    Given A ${volume_size_gb} GB ${volume_type} volume with 3 replicas
+    And Attach volume to node 1
+    And Write data into mount point
```

Comment:
This actually depends on the contributor of this test case definition. Contributors will write/express it subconsciously in a way that they think it's natural. 

So our idea goal is without any hard guide, the test case definition written by contributors can involuntarily automatically roughly fit into our implemented keywords and we achieve the goal that automate a test case with the least effort.

Reply:
IIUC, I agree that different contributors may use various expressions to clarify the same matter in the future. Currently, the progress of this PR involves using the same name in `.resource` to describe the test case and implementing the functionality within it. I'll look further into how to do this. Can you give me more relevant information or tips?

Reply:
We can walk through our existing test cases in the `tests` folder to know how people describe a test case before, so we know what keywords we can provide beforehand.

Reply:
Thanks for your advice. I will refer to it to sort out the commonly used keywords. If it takes a little effort, I will create a ticket first.

Reply:
https://github.com/longhorn/longhorn/issues/6424